src/stores/utxostore.ts

```diff
@@ -26,14 +26,22 @@ export const utxoStore = readable<UtxoStore>(initialState, function (set) {
 
   // trick to update the store using updaters + set
   const updateState = (updater: Updater<UtxoStore>) => {
-    console.log('updating state')
     const newState = updater(state);
     state = newState;
+    console.log('updating state to', state);
     set(state);
   }
 
   detectProvider('marina')
     .then((marina: MarinaProvider) => {
+      marina.getNetwork().then(console.log);
```

Comment:
This can be a check needed for #15

For now lets remove

Reply:
Ahhh, damn, I was preparing for #15 and forgot this line in there. Will remove.